packages/utils-types/README.md

```diff
@@ -0,0 +1,18 @@
+# Types utils (`@hint/utils-types`)
+
+Set of common types used in webhint.
+
+## Installation
+
+To install the package, you need to run:
+
+```bash
+npm install @hint/utils-types
+```
```

Comment:
Use the same language than other packages:

> This package is installed automatically by webhint:
> 
> `npm install hint --save-dev`

I guess this applies to all the other packages you have in another PR.

Reply:
done!

Yep, I will need to update the other packages in the other PR :)